LICENSE

```diff
@@ -217,3 +200,199 @@ solely pursuant to the terms of the relevant commercial agreement.
+
+
+================================================================================
+
+Meyer CSS reset
+http://meyerweb.com/eric/tools/css/reset/
+License: none (public domain)
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+normalize.css
+http://github.com/necolas/normalize.css
+Copyright (c) Nicolas Gallagher and Jonathan Neal
+License: https://github.com/necolas/normalize.css/blob/master/LICENSE.md
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+CSS3-mediaqueries.js
+http://code.google.com/p/css3-mediaqueries-js/
+License: MIT License, http://www.opensource.org/licenses/mit-license.php
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+jQuery
+Copyright 2012 jQuery Foundation and other contributors
+http://jquery.com
+License: MIT License, http://jquery.org/license/
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+Mustache.js
+Copyright (c) 2009 Chris Wanstrath (Ruby)
+Copyright (c) 2010 Jan Lehnardt (JavaScript)
+http://mustache.github.com
+License: MIT License
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+Raphael
+http://raphaeljs.com, from fork at git@github.com:mobz/g.raphael.git
+Copyright (c) 2010 Dmitry Baranovskiy
+License: MIT (http://raphaeljs.com/license.htm
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+date-range-parser
+https://github.com/mobz/date-range-parser
+Contributed to the Apache Software Foundation by Ben Birch - Aconex
+License: Apache License v2 (http://www.apache.org/licenses)
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
+
+================================================================================
+
+For the `blackbox/docs/_static/bootstrap*` files:
+
+Bootstrap
+https://getbootstrap.com
+Copyright 2012 Twitter, Inc.
+License: Apache License v2 (http://www.apache.org/licenses)
+
+FIXME: can probably be removed?
```

Comment:
it's still used in the crate-admin https://github.com/crate/crate-admin/blob/edce34f00eb526f75dda60ae0a1c18cf147c3e08/package.json#L62